Server/controllers/Common/container.js

```diff
@@ -287,11 +288,27 @@ exports.getItemsWithFilters = async (
         description: true,
         categories: true,
       },
-      orderBy: {
-        [sortBy]: orderBy,
-      },
     });
+
+    const availableItems = items.filter(item => item.available);
+    const unavailableItems = items.filter(item => !item.available);
+
+    availableItems.sort((a, b) => {
+      return isAscending ? a[sortBy] - b[sortBy] : b[sortBy] - a[sortBy];
+    });
+
+    unavailableItems.sort((a, b) => {
+      return isAscending ? a[sortBy] - b[sortBy] : b[sortBy] - a[sortBy];
+    });
+
+    if (!isAvailable) {
```

Comment:
It doesn't make any sense to sort the article list by the "isAvailable" boolean. If "isAvailable" is true, then return only the available articles, otherwise, return all articles

Reply:
it does, because it will return the available articles first if available is true, and after, the unavailable articles.